### PR TITLE
Fix method on_tick will not update metric when enable_auto_tune is false

### LINKED ISSUE #11787: Metric `BACKUP_SOFTLIMIT_GAUGE` should always be updated in method `on_tick`
### What version of TiKV are you using?

`Master` branch

### What did happened?

image

### PR DESCRIPTION
**Pull request by yuqi1129**
### What is changed and how it works?

Close #11787 

What's Changed:

In method `on_tick`, we will update the metric `BACKUP_SOFTLIMIT_GAUGE` no matter `enable_auto_tune` is true or not

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- PR to update `pingcap/tidb-ansible`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- No code

Side effects

- Performance regression
    - Consumes more CPU
    - Consumes more MEM
- Breaking backward compatibility

### Release note 

```release-note
None.
```

### CODE REVIEW

File: components/backup/src/endpoint.rs [hunk 1]
```diff
@@ -586,23 +586,22 @@ impl SoftLimitKeeper {
...
 
-        let quota_val = cpu_quota
-            .get_quota(|s| s.contains("bkwkr"))
-            .clamp(1, num_threads);
         self.limit.resize(quota_val).await.map_err(|err| {
-            warn!("error during appling the soft limit for backup."; "err" => %err);
+            warn!("error during appling the soft limit for backup.";
+                "current_limit" => %self.limit.current_cap(),
+                "to_set_value" => %quota_val,
+                "err" => %err);
```
Comment (tabokie) [inline components/backup/src/endpoint.rs:601]:
```suggestion
            warn!(
                "error during appling the soft limit for backup.";
                "current_limit" => %self.limit.current_cap(),
                "to_set_value" => %quota_val,
                "err" => %err,
            );
```

**Review by tabokie — APPROVED** [review]

**Review by kennytm — APPROVED** [review]

File: components/backup/src/endpoint.rs (hunk 1 above)
Comment (yuqi1129) [reply components/backup/src/endpoint.rs:601]: 👍

File: components/backup/src/endpoint.rs [hunk 2]
```diff
@@ -586,23 +586,23 @@ impl SoftLimitKeeper {
...
+                .get_quota(|s| s.contains("bkwkr"))
+                .clamp(1, num_threads);
         }
 
-        let quota_val = cpu_quota
-            .get_quota(|s| s.contains("bkwkr"))
-            .clamp(1, num_threads);
         self.limit.resize(quota_val).await.map_err(|err| {
-            warn!("error during appling the soft limit for backup."; "err" => %err);
+            warn!("error during appling the soft limit for backup.";
```
Comment (tabokie) [inline components/backup/src/endpoint.rs:598]:
```suggestion
            warn!(
                "error during appling the soft limit for backup.";
```

**Comment by yuqi1129** [comment]
cargo build  returns 
```

[2022-01-06T08:28:36.243Z] Cannot contact build-6gfl6: hudson.remoting.ChannelClosedException: Channel "hudson.remoting.Channel@c053464:JNLP4-connect connection from 172.16.5.87/172.16.5.87:39784": Remote call on JNLP4-connect connection from 172.16.5.87/172.16.5.87:39784 failed. The channel is closing down or has closed down

[2022-01-06T08:33:28.757Z] Could not connect to build-6gfl6 to send interrupt signal to process
```

**Pull request merged**